### PR TITLE
mutt: add page

### PR DESCRIPTION
**Pull request by bigspennyj**

### CODE REVIEW

File: pages/common/mutt.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`mutt -a {{file1}} {{file2}} -- {{recipient@example.com}}`
+
+- Specify a file to include as the message body:
+
+`mutt -i {{file}} {{recipient@example.com}}`
+
+- Specify a draft file containing the header and the body of the message:
+
+`mutt -H {{file}} {{recipient@example.com}}`
```
Comment (agnivade) [inline pages/common/mutt.md]: How are the header and the body of the message separated in the file ?

Comment (bigspennyj) [inline pages/common/mutt.md]: Headers and body are specified as follows:

```
From: sender@sender.com
To: recipient@example.com
Cc: cc@example.com
Bcc: bcc@example.com
Subject: My awesome message

Here is the body of my message.
```

Comment (agnivade) [inline pages/common/mutt.md]: Please add a link to the format of the draft file in the description.

**Comment by agnivade** [comment]
Hi @bigspennyj - any updates on this ?

**Comment by agnivade** [comment]
Hi @bigspennyj - I am going to close this PR due to inactivity. Feel free to reopen this whenever you get some time.

File: pages/common/mutt.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/mutt.md]: @agnivade after some investigation, I found a reference that could be added here. According to https://en.wikipedia.org/wiki/Email#Header_fields, email headers must conform to IETF's [RFC 5322](https://tools.ietf.org/html/rfc5322). I think we could change the example description to:

`Specify a draft file containing the header and the body of the message, in RFC 5322 format:`

**Comment by waldyrious** [comment]
I'll reopen given my response to the inline comment thread.

**Comment by waldyrious** [comment]
Hey @bigspennyj, if you could please check the "enable edits by maintainers" option, we'll be able to finish this up for you :)

**Comment by waldyrious** [comment]
Unfortunately @bigspennyj seems to be inactive. I don't want to throw out the contribution, though, so I'll do the [manual process](https://help.github.com/articles/checking-out-pull-requests-locally/#modifying-an-inactive-pull-request-locally) (which unfortunately requires closing this PR and opening a new one -- but at least the authorship will be preserved).

**Comment by waldyrious** [comment]
Actually, let's just merge this and change the page afterwards -- the concerns raised in the inline comments are not exactly deal-breakers, just requests for additional clarification.

**Pull request merged**